### ISSUE TITLE
Plan and run a messaging test

### ISSUE THREAD

**Issue by hannahkane** [MozillaFoundation/Advocacy#990]
In parallel with v1 of the style guide, we'll run some messaging tests to hone in on our consumer-facing messages for the campaign. The results of the test will inform v2 of the styleguide, and ultimately the consumer-facing deliverables (website, promotional plan, buyer's guide).

This ticket represents the work of developing creative assets for each of the headlines we're testing, as well as planning, running, and analyzing results of the campaign. @lovegushwa - feel free to write additional tickets as needed.

Jordan to work with Kevin Fann and Alex on this

**Comment by hannahkane** [comment]
@lovegushwa - did you come up with a plan or next step on this? Could you please drop in here so we can all know? Thanks!

**Comment by lovegushwa** [comment]
@hannahkane ... currently working with @beccaklam on message testing [deck.](https://docs.google.com/a/mozilla.com/presentation/d/1NMM_sOxrJIDfeDbNtHmJh_4EZjHk9oQrcuQ54k5BDqY/edit?usp=sharing) 

I think we could start running some tests with Alex (or via email with Will) the week of the 21st.
Will post updates here this week once @beccaklam gets back from PTO.

**Comment by hannahkane** [comment]
@lovegushwa - sounds perfect. As discussed, would be great if you could add a few slides to the deck detailing possible testing plans (e.g. FB w/ landing pages, Twitter, email, focus groups) and the relative advantages/disadvantages of each, so we can discuss. The final testing plan will likely be informed by @tlasadea's work around [audience definition](https://github.com/MozillaFoundation/Advocacy/issues/1002).

**Comment by hannahkane** [comment]
My understanding is we've decided to simply choose a direction for the look and feel, so I'm closing this. Can add a new ticket for audience test work, if needed.

cc @HPaulJohnson to correct me if I'm wrong
